mlir/test/Interfaces/TilingInterface/tile-and-fuse-consumer.mlir

```diff
@@ -265,7 +265,7 @@ module {
         %c4 = arith.constant 4 : index
         %c64 = arith.constant 64 : index
         %c0 = arith.constant 0 : index
-        %1 = scf.forall (%arg3, %arg4) in (2, 2) shared_outs(%arg5 = %arg2) -> (tensor<64x32xf32>) {
+        %1 = scf.forall (%arg3, %arg4) = (0, 0) to (64, 32) step (32, 32) shared_outs(%arg5 = %arg2) -> (tensor<64x32xf32>) {
```

Comment:
Can you elaborate a bit why this tests the case? I'm confused that why no additional tests are added.

Reply:
This test also tests the case, because offsets are dynamic, but it is not entirely clear in this case because the unpack is perfectly aligned (i.e., the dynamic result sizes could potentially be optimized away). I will add another test with an imperfectly aligned unpack op.